src/components/Calculator.css

```diff
@@ -4,9 +4,14 @@
   padding: 0;
 }
 
+.calculator {
+  display: grid;
+  width: 50em;
+  margin: auto;
+}
+
 .calculator-cont {
   display: grid;
-  height: 962px;
   width: 50em;
   margin: auto;
```

Comment:
Just as the previous reviewer asked, your calculator height spans the viewport height and does not look great as the template given and it also causes scrolling. But I have made some changes in a few areas you should modify your code to get things right 👍

- [ ] Kindly have a margin set to `margin: 50px auto;`. You can replace the margin you had earlier 👍

Also in the class below this - That's the `.result{}` class. Kindly work on your code to match something like this below:
```mdx
.result {
  text-align: right;
  color: white;
  font-size: 3.5em;
  margin: 0;
  padding-top: 10px;
  padding-right: 10px;
  background-color: rgb(136, 134, 149);
  height: 100px;
}
```